helpdesk/teams/migrations/0005_alter_team_options_teamattendanceevent.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 4.2.11 on 2024-12-24 18:40
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ("teams", "0004_add_team_comment"),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name="team",
+            options={"ordering": ["tla"]},
+        ),
+        migrations.CreateModel(
+            name="TeamAttendanceEvent",
+            fields=[
+                ("id", models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                (
+                    "type",
+                    models.TextField(
+                        choices=[("AR", "Arrived"), ("LE", "Left"), ("DE", "Delayed"), ("DO", "Dropped Out")],
```

Comment:
Nit: I think this migration needs regenerating based on the current text choices